packages/ddp/livedata_connection.js

```diff
@@ -520,12 +532,41 @@ _.extend(Connection.prototype, {
         if (!existing.ready)
           existing.readyCallback = callbacks.onReady;
       }
+
+      // XXX COMPAT WITH 1.0.3.1 we used to have onError but now we call
+      // onStopped with an optional error argument
       if (callbacks.onError) {
         // Replace existing callback if any, so that errors aren't
         // double-reported.
         existing.errorCallback = callbacks.onError;
       }
+
+      if (callbacks.onStopped) {
+        existing.stoppedCallback = callbacks.onStopped;
+      }
+
+      // Return the exact same handle so that we can use === to check if the
```

Comment:
I'm open to either one. It seems weird to generate lots of identical handles on the one hand, on the other hand the `id` solution could be less magical and easier to document. Maybe we should do both? Basically don't generate lots of unnecessary handles as an optimization but also put the `id` on there, document that change, and use it everywhere.

What do you think?